src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+type Colors = 'red' | 'green' | 'blue';
 
+function getTwoDecimalPlaces(area:number): number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shapes,
+  color: Colors,
+  getArea(): number,
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  public shape: Shapes = 'triangle';
 
+  constructor(
+    public color: Colors,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if ((this.a <= 0 || this.b <= 0 || this.c <= 0)
+    || (this.a >= this.b + this.c
+    || this.b >= this.a + this.c
+    || this.c >= this.a + this.b)) {
+      throw new Error('ERROR - not a triangle');
```

Comment:
This condition should be broken into two separate ones and given a more informative error text.